src/ios/ChromeProxy.m

```diff
@@ -0,0 +1,93 @@
+/*
+ Implementation for ChromeProxy
+ 
+ ChromeProxy receives the get, set, and clear commands from Cordova
+ to enable and disable proxying.
+ */
+
+#import "ChromeProxy.h"
+#import "ChromeProxyURLProtocol.h"
+#import <Cordova/CDVPluginResult.h>
+#import <Cordova/CDVUserAgentUtil.h>
+
+@implementation ChromeProxy
+
+static NSString *LOG_TAG = @"[ChromeProxy]";
+static NSDictionary *proxyDictionary;
+
++ (void)load {
+    NSLog(@"%@ Registering nsurlprotocol class", LOG_TAG);
+    [NSURLProtocol registerClass:[ChromeProxyURLProtocol class]];
+}
+
++ (void)setProxy:(NSString *)scheme withHost:(NSString *)host withPort:(NSNumber *)port {
+    proxyDictionary = @{
+                        (NSString *)[NSString stringWithFormat:@"%@Enable", scheme]:[NSNumber numberWithInt:1],
+                        (NSString *)[NSString stringWithFormat:@"%@Proxy", scheme]:host,
+                        (NSString *)[NSString stringWithFormat:@"%@Port", scheme]:port,
```

Comment:
+[NSString stringWithFormat:] returns an NSString*; I don't understand your cast here and in the previous two lines.